frameworklauncher/src/main/java/com/microsoft/frameworklauncher/applicationmaster/Node.java

```diff
@@ -75,6 +92,7 @@ public int compareTo(Node other) {
     if (thisAvailableResource.getMemoryMB() < otherAvailableResource.getMemoryMB()) {
```

Comment:
>getMemoryMB [](start = 30, length = 11)

you remove CPU and Mem without breaking the UTs? maybe you should add UTs for the two cases.

Reply:
Previous UT didn't cover memory and cpu

---
In reply to: [200578700](https://github.com/Microsoft/pai/pull/822#discussion_r200578700) [](ancestors = 200578700)

Reply:
do you add the UT to cover memory and cpu? I didn't find the new UT?

---
In reply to: [200593832](https://github.com/Microsoft/pai/pull/822#discussion_r200593832) [](ancestors = 200593832,200578700)

Reply:
the last UT, which available resource and total resource are the  same ,compare the memory and cpu